compose.yml

```diff
@@ -141,3 +141,5 @@ volumes:
 networks:
   neosync-network:
     name: neosync-network
+  temporal-network:
+    name: temporal-network
```

Comment:
This shouldnt be needed here because it is already included with the statement at the top:

```
  - path: ./compose/temporal/compose.yml
    env_file:
      - ./compose/temporal/.env
```

Did you run in to an issue by not having this defined here?

Reply:
You are right, it's not needed, the problem was with the docker compose version, I was using `v2.19.1` which does not support `include`, so upgrading the docker-compose version to the latest (v2.28.1-desktop.1) solved the issue, Thanks